rpcs3/Emu/Cell/Modules/cellSaveData.cpp

```diff
@@ -89,7 +90,10 @@ namespace
 
 vm::gvar<savedata_context> g_savedata_context;
 
-std::mutex g_savedata_mutex;
+struct savedata_mutex
+{
+	std::mutex mutex;
```

Comment:
std::mutex may not support it... It needs a semaphore<> because it can be left in any state.